test/test-real-values.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+const path = require('path');
+const compareVersions = require('compare-versions');
+const chalk = require('chalk');
+
+/** @type {Record<string, string[]>} */
+const browsers = {
```

Comment:
Can we rename this to `blockList`? I think `browsers` is such an overloaded term in this context already.